src/main/java/com/hyperlocal/server/MerchantController.java

```diff
@@ -0,0 +1,73 @@
+package com.hyperlocal.server;
+
+import java.util.Arrays;
+import java.util.concurrent.CompletableFuture;
+
+import com.github.jasync.sql.db.Connection;
+import com.github.jasync.sql.db.QueryResult;
+import com.github.jasync.sql.db.mysql.MySQLConnectionBuilder;
+import com.google.gson.Gson;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+// Rest Controller for Merchants
+
+@RestController
+public class MerchantController {
+
+  private static final String DATABASE_URL = "jdbc:mysql:///hyperlocal?socketFactory=com.google.cloud.sql.mysql.SocketFactory&cloudSqlInstance=speedy-anthem-217710:us-central1:hyperlocal";;
+  private static Connection connection;
+  private static final String MERCHANT_UPDATE_PREPARED_STATEMENT = "UPDATE `Merchants` SET `MerchantName` = ?, `MerchantPhone` = ? WHERE `MerchantID`= ?;";
```

Comment:
naming is just MERCHANT_UPDATE_STATEMENT is enough since all statements are going to be prepared statements only